catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -30,4 +38,128 @@
+            <list>
+                <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+            </list>
+        </argument>
+    </bean>
+
+
+    <bean id="jpegMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="fallback.jpeg"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+            </list>
+        </argument>
+    </bean>
+
+
+    <bean id="mp4MetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="fallback.mp4"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+            </list>
+        </argument>
+    </bean>
+
+
+    <bean id="mpegMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="fallback.mpeg"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+            </list>
+        </argument>
+    </bean>
+
+
+    <bean id="officeDocMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="fallback.doc"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+            </list>
+        </argument>
+    </bean>
+
+
+    <bean id="pdfMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
```

Comment:
The existing PdfMetacardType is defined in blueprint and exported as an OSGi service here: https://github.com/codice/ddf/blob/master/catalog/transformer/catalog-transformer-pdf/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L31